lib/src/openApiToZod.ts

```diff
@@ -265,8 +270,7 @@ const getZodChainableDefault = (schema: SchemaObject) => {
     if (schema.default) {
         const value = match(schema.type)
             .with("number", "integer", () => unwrapQuotesIfNeeded(schema.default))
-            .with("array", "object", () => JSON.stringify(schema.default))
-            .otherwise(() => (typeof schema.default === "string" ? `"${schema.default}"` : schema.default));
+            .otherwise(() => JSON.stringify(schema.default));
         return `default(${value})`;
     }
```

Comment:
Unrelated to the PR (can put it in a separate one if preferred), but I noticed that the schema "type" prop is optional, or can be an union, etc... and that `JSON.stringify` should work for any value shape AFAIK.